$OpenBSD$

Use xmltv rather than xmltv-version for docs.
Remove dvb linux specific.

--- Makefile.PL.orig	Thu Jun 14 07:59:46 2012
+++ Makefile.PL	Fri Jun 15 00:28:43 2012
@@ -79,7 +79,7 @@ my $location = '$(DESTDIR)/./$(PREFIX)';
 our %extra_constants;
 %extra_constants
   = (INST_PLAINDOC   => 'blib/doc',
-     INSTALLPLAINDOC => "$location/share/doc/xmltv-$::VERSION",
+     INSTALLPLAINDOC => "$location/share/doc/xmltv",
      INST_SHARE      => 'blib/share',
      INSTALLSHARE    => "$location/share/xmltv",
 
@@ -454,22 +454,22 @@ my @opt_components
        grab_need_share => [ 'it' ],
      },
 
-     { name => 'tv_grab_it_dvb',
-       blurb => 'Grabber for Italy from DVB-S stream',
-       exes => [ 'grab/it_dvb/tv_grab_it_dvb' ],
-       pl_files => { 'grab/it_dvb/tv_grab_it_dvb.PL' => 'grab/it_dvb/tv_grab_it_dvb' },
-       share_files => { 'grab/it_dvb/channel_ids' => 'tv_grab_it_dvb/channel_ids',
-                        'grab/it_dvb/sky_it.dict' => 'tv_grab_it_dvb/sky_it.dict',
-                        'grab/it_dvb/sky_it.themes' => 'tv_grab_it_dvb/sky_it.themes',
-                   },
-       to_clean => [ 'grab/it_dvb/tv_grab_it_dvb' ],
-       deps => [ 'grab/it_dvb/tv_grab_it_dvb' => [ 'grab/it_dvb/tv_grab_it_dvb.in' ] ],
-       grab_need_share => [ 'it_dvb' ],
-       prereqs => { 'Data::Dump' => 0,
-                    'IO::Select' => 0,
-                    'Linux::DVB' => 0,
-                    'Time::HiRes' => 0 } ,
-     },
+#     { name => 'tv_grab_it_dvb',
+#       blurb => 'Grabber for Italy from DVB-S stream',
+#       exes => [ 'grab/it_dvb/tv_grab_it_dvb' ],
+#       pl_files => { 'grab/it_dvb/tv_grab_it_dvb.PL' => 'grab/it_dvb/tv_grab_it_dvb' },
+#       share_files => { 'grab/it_dvb/channel_ids' => 'tv_grab_it_dvb/channel_ids',
+#                        'grab/it_dvb/sky_it.dict' => 'tv_grab_it_dvb/sky_it.dict',
+#                        'grab/it_dvb/sky_it.themes' => 'tv_grab_it_dvb/sky_it.themes',
+#                   },
+#       to_clean => [ 'grab/it_dvb/tv_grab_it_dvb' ],
+#       deps => [ 'grab/it_dvb/tv_grab_it_dvb' => [ 'grab/it_dvb/tv_grab_it_dvb.in' ] ],
+#       grab_need_share => [ 'it_dvb' ],
+#       prereqs => { 'Data::Dump' => 0,
+#                    'IO::Select' => 0,
+#                    'Linux::DVB' => 0,
+#                    'Time::HiRes' => 0 } ,
+#     },
 
      { name     => 'tv_grab_in',
        blurb => 'Grabber for India (zipazap.com)',
